src/cmdlib.sh

```diff
@@ -293,6 +293,9 @@ EOF
     fi
     if [ -d "${ovld}" ]; then
         for n in "${ovld}"/*; do
+            if ! [ -d "${n}" ]; then
```

Comment:
Lets warn that these are getting ignored.

Reply:
I don't want to emit a warning for every build though; it's pretty common for `.d` directories to have an implicit pattern match and ignore others.

If you drop a  `README.md` in `/etc/systemd/system`, systemd doesn't warn.

Maybe an alternative (/extension) of this though would be to ignore `README*` in the individual overlay directories too.

Reply:
hrmmm. Ok. Lets just document in the readme that non-directories are ignored (_after_ symlink resolution)

Reply:
it currently says `Each subdirectory of the `overlay.d` directory is added to the OSTree commit, in lexicographic order`. Could possibly be a little better if it also stated that regular files directly below the overlay.d directory are ignored.